fs_storage/models/fs_storage.py

```diff
@@ -339,6 +339,9 @@ def _get_fs_options(self):
             return options
         return self._eval_options_from_env(self.json_options)
 
+    def _get_marker_file_name(self):
+        return "odoo_fs_storage_%s.marker" % self.id
```

Comment:
```suggestion
        return ".odoo_fs_storage_%s.marker" % self.id
```
and drop the manipulation below